src/generators.rs

```diff
@@ -23,14 +23,13 @@ use pyo3::Python;
 use super::{digraph, graph, StablePyGraph};
 use rustworkx_core::generators as core_generators;
 
-/// Generate an undirected cycle graph
+/// Generate an undirected cycle graph.
 ///
 /// :param int num_nodes: The number of nodes to generate the graph with. Node
 ///     weights will be None if this is specified. If both ``num_nodes`` and
-///     ``weights`` are set this will be ignored and ``weights`` will be used.
-/// :param list weights: A list of node weights, the first element in the list
```

Comment:
I believe a “center” node does not really make sense here, nor in the path graphs below.